src/loader/mod.rs

```diff
@@ -409,6 +420,94 @@ pub fn load_cmdline<M: GuestMemory>(
     Ok(())
 }
 
+#[cfg(feature = "image")]
+#[cfg(target_arch = "aarch64")]
+/// ARM64 Image (PE) format support
+pub struct Image;
+
+#[cfg(feature = "image")]
+#[cfg(target_arch = "aarch64")]
+#[allow(missing_docs)]
```

Comment:
Please document the struct and its fields instead.
I also think it would be better to separate `x86_64` and `aarch64` into different modules and export each mod as needed.

Reply:
The struct should not be marked as public. Just put it in `load`.